hicommonwealth/commonwealth
# deleting thread now redirects to discussions

## Link to Issue
Closes: #8744 

## Description of Changes
- User is now redirected to `/discussions` after deleting a thread from within the thread

## "How We Fixed It"

-created a new function `getThreadUrlPath` to replace the use of `getProposalUrlPath` that redirects to `/discussions` instead of `/proposal/${id}` if `type === ProposalType.Thread` is false
NOTE: I changed `getProposalUrlPath` to `getThreadUrlPath` in `DiscussionsPage.tsx` as well as `ViewThreadPage.tsx` to stay consistent with coding intent, despite it not being needed in `DiscussionsPage.tsx` 
## Test Plan
- go to a thread you create or create a thread
- while on `/discussions` page delete the thread
- confirm that thread was deleted and that you are still on `/discussions`
- go to another thread you created or create a new thread
- click into that thread and delete the thread from there
- confirm that you are now redirected to `/discussions`
